models/users.js

```diff
@@ -170,8 +170,6 @@ const fetchPaginatedUsers = async (query) => {
         ...doc.data(),
         phone: undefined,
         email: undefined,
-        tokens: undefined,
-        chaincode: undefined,
```

Comment:
Why are these been removed here?

Reply:
In the previous comment you told "I think we should remove token and chain code also
REF" so I removed them